java/com/google/location/lbs/activity/audioset/dataprocessing/CsvParser.java

```diff
@@ -0,0 +1,244 @@
+import java.io.File;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Arrays;
+
+/**
+ * CSV Class
+ *
+ * <p>CSV is a class for the reading of a dataset in a csv format.
+ *
+ * @author Carver Forbes
+ * @version 0.1
+ */
+public class CsvParser {
+
+  private File file;
+  private ArrayList<String> ytidList;
+  private HashMap<String, IncompleteExample.Example> idToProtoMap;
+  private HashMap<String, String> idToUrlMap;
+
+  private final boolean debug = true;
+
+  /**
+   * CSV constructor
```

Comment:
ditto. o/w this could be interpreted as something that constructs CSV output

Reply:
Ah. Thank you.